ValidationLibrary/Rules/HasNewestPtcsJenkinsLibRule.cs

```diff
@@ -19,18 +19,22 @@ public class HasNewestPtcsJenkinsLibRule : IValidationRule
         private const string JenkinsFileName = "Jenkinsfile";
 
         private const string LibraryName = "jenkins-ptcs-library";
+        private readonly string BranchName = $"feature/{LibraryName}-update";
+        private readonly string PullRequestTitle = $"[Automatic Validation] Update {LibraryName} to latest version.";
         private const string FileMode = "100644";
 
         public string RuleName => $"Old {LibraryName}";
 
         private readonly Regex _regex = new Regex($@"[""']{LibraryName}@(\d+.\d+.\d+.*)[""']", RegexOptions.Compiled | RegexOptions.IgnoreCase);
 
-        private readonly ILogger _logger;
+        private readonly ILogger<HasNewestPtcsJenkinsLibRule> _logger;
+        private readonly GitUtils _gitUtils;
```

Comment:
Have you considered defining an interface for GitUtils? Utils also sounds like a static helper class.

Reply:
I was kind of thinking about refactoring it and the whole `Fix` section to use wrappers which would be easier to understand , because the logic is kind of simple here, but the actual implementation is somewhat complicated.

I added a separate issue for this https://github.com/protacon/repository-validator/issues/115